source/documentation/deploying_services/index.md

```diff
@@ -6,7 +6,11 @@ Many 12-factor applications rely on backing services such as a database, an emai
 
 In Cloud Foundry, backing and routing services are referred to as 'services' and are available through the Cloud Foundry ``cf marketplace`` command.
 
-Currently, the only services available in the marketplace are the backing [PostgreSQL database service](/#using-postgresql) and [MySQL database service](/#using-mysql).
+Currently, the following database services are available in the marketplace:
+[PostgreSQL](/#using-postgresql)
+[MySQL](/#using-mysql)
+MongoDB
```

Comment:
We would have to do this in two steps. 1. Merge Mysql + Postgres to make the docs more generic. 2 Add Mongo specific changes. This PR can be first step but we would have to remove all mongo stuff.